### PR TITLE
Add transaction info to link

### PR DESCRIPTION
**Pull request by phieronymus**
Linked to https://github.com/univapaycast/services-typescript/pull/800

### CODE REVIEW

File: src/resources/common/Configuration.ts [hunk 1]
```diff
@@ -117,6 +121,7 @@ export interface ConfigurationItem {
     securityConfiguration: SecurityConfiguration;
     subscriptionConfiguration: SubscriptionsConfiguration;
     userTransactionsConfiguration?: UserTransactionsConfiguration;
+    widgetConfiguration?: WidgetConfiguration | null;
```
Comment (perrin4869) [inline src/resources/common/Configuration.ts:124]: sorry for the confusion, but I went with `checkoutConfiguration` as the name here!

File: src/resources/common/Configuration.ts [hunk 2]
```diff
@@ -95,6 +95,10 @@ export interface DescriptorProvidedConfiguration {
     name: string;
     phoneNumber: string;
 }
+export interface WidgetConfiguration {
+    ecEmail?: { enabled?: boolean | null } | null;
```
Comment (perrin4869) [inline src/resources/common/Configuration.ts:99]: I think none of these is nullable or optional

**Comment by perrin4869** [comment]
Also linked https://github.com/univapaycast/gyron-payments-api/pull/4614

**Comment by phieronymus** [comment]
@perrin4869 Thanks for catching the issues 🙂 I was trying with the ticket values.

**Review by perrin4869 — APPROVED** [review]

**Pull request merged**